addons/project/static/tests/tours/project_sharing_tour.js

```diff
@@ -115,3 +115,24 @@ registry.category("web_tour.tours").add("portal_project_sharing_tour", {
         return projectSharingSteps.slice(projectSharingStepIndex, projectSharingSteps.length);
     }
 });
+
+registry.category("web_tour.tours").add("project_sharing_with_blocked_task_tour", {
+    test: true,
+    url: "/my/projects",
+    steps: () => [{
+        trigger: 'table > tbody > tr a:has(span:contains("Project Sharing"))',
+        content: 'Click on the portal project.'
+    }, {
+        trigger: 'iframe .o_kanban_record',
+        run: 'click',
+        content: 'Click on the task'
+    }, {
+        trigger: 'iframe a:contains("Blocked By")',
+        run: 'click',
+        content: 'Go to the Block by task tab',
```

Comment:
Is the `run: 'click',` necessary here ? because it is the default action thus I believe we can omit it.